Makefile

```diff
@@ -324,6 +330,16 @@ ut combined.coverprofile: vendor/.up-to-date $(FELIX_GO_FILES)
 	@echo Running Go UTs.
 	$(DOCKER_GO_BUILD) ./utils/run-coverage
 
+bin/fv.test: vendor/.up-to-date $(FELIX_GO_FILES)
+	$(DOCKER_GO_BUILD) go test ./fv -c --tags fvtests -o bin/fv.test
```

Comment:
I saw a neat trick in that Cloudflare article: using a build tag to identify which tests to build/run and I went for that approach.  WDYT?